src/virtio-net.c

```diff
@@ -0,0 +1,284 @@
+#include <assert.h>
```

Comment:
Remove `<assert.h>` since it is not used.

Reply:
Already removed it.